x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/expanded_row_json_pane.tsx

```diff
@@ -19,14 +19,17 @@ export const ExpandedRowJsonPane: FC<Props> = ({ json, dataTestSubj }) => {
     <EuiFlexGroup data-test-subj={dataTestSubj}>
       <EuiFlexItem>
         <EuiSpacer size="s" />
-        <EuiCodeEditor
-          value={JSON.stringify(json, null, 2)}
-          readOnly={true}
-          mode="json"
+        <EuiCodeBlock
```

Comment:
It's worth adding an `aria-label` for this, something like `JSON of data frame analytics configuration`